### ISSUE TITLE
[Feature Request] Firefox plugin?

### ISSUE THREAD

**Issue by mcampbell** [humanloop/memorai#15]
The Chrome plugin works great; would it be possible to do a Firefox one also?

**Comment by xohathcb** [comment]
Yeah I would also like to see a Firefox version

**Comment by Jfaler** [comment]
I agree that a Firefox extension would be great and I wonder if the creators are willing to take donations to make it possibly happen :)
